Shared/Samples/Set spatial reference/SetSpatialReferenceView.swift

```diff
@@ -0,0 +1,47 @@
+//  Copyright © 2024 Esri. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SetSpatialReferenceView: View {
+    /// A map with spatial reference and a basemap.
+    @State private var map: Map = {
+        let map = Map(spatialReference: .worldBonne)
+        map.basemap = Basemap(
+            baseLayer: ArcGISMapImageLayer(url: .worldCities)
+        )
+        return map
+    }()
+    
+    var body: some View {
+        MapView(map: map)
+    }
+}
+
+private extension SpatialReference {
+    /// The spatial reference for the sample World Bonne (WKID: 54024).
```

Comment:
```suggestion
    /// The spatial reference for the sample, World Bonne (WKID: 54024).
```

Not sure about this one.... Dose this need a comma here?

Reply:
Yes, thank you for the feedback! I appreciate your input